modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -24,6 +24,23 @@ use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
 use Drupal\oe_media_iframe\Plugin\media\Source\Iframe;
 use Drupal\paragraphs\Entity\Paragraph;
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ *
+ * Adds a bare, markup-free template suggestion to all paragraph fields.
+ */
+function oe_bootstrap_theme_paragraphs_theme_suggestions_field_alter(array &$suggestions, array $variables): void {
+  $element = $variables['element'];
+
+  // Do not output field labels and wrapping markup for paragraph fields.
+  if (isset($element['#entity_type']) && $element['#entity_type'] === 'paragraph') {
+    // Prepend the new suggestion to the list. This will put it right after the
+    // default field template. By doing this we allow to override single
+    // fields, while keeping all the rest markup-free.
+    array_unshift($suggestions, 'field__bare');
```

Comment:
@drishu we brought this from OE Theme, we understand is necessary in OE Bootstrap theme. Right?

Another question is that it could be good to add an assertion in the test at EntityReferenceRevisionTest. We must ensure this suggestion is applied and no labels or attributes are added (because twig field__bare is used.). Is correct this assumption?

Reply:
yes, field__bare is good.
"ensure this suggestion is applied" since we check there for my-4 and find it, it means that the suggestion is applied, and that will be enough for now